### PR TITLE
Update configuration manual

### PR DESCRIPTION
**Pull request by LenaLenaPan**

### CODE REVIEW

File: docs/en_US/config-description/.md
Comment (lee-emqx) [inline docs/en_US/config-description/.md:1]: What's this?

File: docs/en_US/config-description/acl.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+  These parameters can include variables like: 
+
+  - `%A`: Permission to be verified: `1` for subscription and `2` for publish
+  - `%u`: Username
+  - `%c`: MQTT Client ID
+  - `%a`: Client's network IP address
+  - `%r`: The protocol used by the client can be:mqtt, mqtt-sn, coap, lwm2m and stomp
+  - `%m`: Mount point
+  - ``%t`: Topic
```
Comment (lee-emqx) [inline docs/en_US/config-description/acl.md:187]: Typo?

File: etc/nanomq_example.conf [hunk 2]
```diff
@@ -323,7 +323,7 @@ auth {
 	# # allow anonymous login
 	# #
 	# # Hot updatable
-	# # Value: true | false
+	# # Value: true | falsexia'mi
```
Comment (LenaLenaPan) [inline etc/nanomq_example.conf:326]:
```suggestion
	# # Value: true | falsexia
```

Comment (LenaLenaPan) [inline etc/nanomq_example.conf]:
```suggestion
	# # Value: true | false
```

File: docs/en_US/config-description/acl.md (hunk 1 above)
Comment (LenaLenaPan) [inline docs/en_US/config-description/acl.md:187]:
```suggestion
  - `%t`: Topic
```

Comment (LenaLenaPan) [reply docs/en_US/config-description/acl.md]: solved the issue, thanks

**Review by RanMaoyi — APPROVED** [review]

File: docs/en_US/config-description/gateway.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+- `port`: Specifies the port number for the HTTP server. The value should be between 0 and 65535.
+- `parallel`: Specifies the maximum number of concurrent requests that the HTTP server can handle.
+- `username`: Specifies the username for basic authorization when accessing the HTTP server.
+- `password`: Specifies the password for basic authorization when accessing the HTTP server.
+
+
+
+## SOME/IP Gateway
+
+NanoMQ now supports SOME-IP data communication based on the AUTOSAR standard via the SOME/IP Gateway.
```
Comment (JaylinYu) [inline docs/en_US/config-description/gateway.md:101]: SOME/IP

File: docs/en_US/config-description/introduction.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 # Configuration
 
-NanoMQ configuration files come in two versions: the Classic KV version and the HOCON version. The HOCON version is a new configuration format introduced in version 0.14. The KV version is the original configuration file format of NanoMQ, which will be retained and compatible with older users for the long term (although some new features will only support the new HOCON configuration format).
+NanoMQ has incorporated HOCON syntax support since version 0.14. However, there have been syntax updates introduced in version 0.19. For this reason, users of NanoMQ versions 0.14 to 0.18 should consult the [NanoMQ 0.14-0.18 Configuration Guide](https://chat.openai.com/c/v014.md) for specifics on the appropriate syntax.
+
+While NanoMQ continues to support the traditional Key-Value (KV) configuration syntax, it's important to note that some new features will only be compatible with the HOCON configuration format. For details on the KV syntax, please refer to the [Classic KV Format Configuration Guide](https://chat.openai.com/c/v013.md).
+
+The focus of this chapter is to provide guidance on configuring NanoMQ using the updated HOCON syntax.
```
Comment (JaylinYu) [inline docs/en_US/config-description/introduction.md:7]: Make it bold text?

File: docs/en_US/config-description/logs.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  }
+}
+```
+
+## **Configuration Items**
+
+- `to`: Specifies the destination(s) where the logs will be emitted. Optional values:
+  - `file`: Write logs to a file.
+  - `console`: Write logs to standard I/O
+  - `syslog`: Wwrite logs to syslog
```
Comment (JaylinYu) [inline docs/en_US/config-description/logs.md:25]: double W

**Review by JaylinYu — APPROVED** [review]

**Comment by JaylinYu** [comment]
Hope this could make everyone happy

**Pull request merged**